static/backbone/js/templates/prints/printEditor.html

```diff
@@ -0,0 +1,246 @@
+                </div>
+                <div class="clearfix">
+                    <div id="holder2" class="holder">
+                        <table class="paper_form" id="paper_form"></table>
+                    </div>
+                </div>
+                <div class="clearfix">
+                    <div class="instructions" id="instructions_slave" style="float:left;width:654px;">&nbsp;</div>
+                    <img src="http://chart.apis.google.com/chart?cht=qr&amp;chld=Q|0&amp;chl=lg&amp;chs=75x75" id="qrcode">
+                </div>
+            </div>
+            
+            <div class="clearfix hidden-form" style="display:block;position:absolute;left:2500px;">
+                <span>width:</span><input name="width" value="{{ width }}" /><br>
+                <span>height:</span><input name="height" value="{{ height }}" /><br>
+                <span>zoom:</span><input id="zoom" name="zoom" value="{{ zoom }}" /><br>
+                <span>center_lat:</span><input id="center_lng" name="center_lng" value="{{ center_lng }}" /><br>
+                <span>center_lng:</span><input id="center_lat" name="center_lat" value="{{ center_lat }}" /><br>
+                <span>layer_ids:</span><input name="layer_ids" value="{{ layer_id_string|default_if_none:"" }}" /><br>
+                <span>scan_ids:</span><input name="scan_ids" value="{{ scan_id_string|default_if_none:"" }}" /><br>
+                <span>map_provider:</span><input type="text" name="map_provider" value="{{ basemap_id }}" /><br>
+                <span>generate_pdf:</span><input type="text" name="generate_pdf" value="on" /><br>
+                <span>layout:</span><input type="text" name="layout" id="layout" value="{{selectedLayout}}" /><br>
+                <span>project_id:</span><input type="text" name="project_id" id="project_id" value="{{selected_project.id}}" />
+            
+				<table id="tbl">
+					<tr>
+						<td>Field</td>
+						<td>Width</td>
+						<td>Ordering</td>
+						<td>ID</td>
+					</tr>
+					{% for form in formset.forms %}
+						<tr id="{{ form.prefix }}-row">
+							<td {% if form.field.errors or form.non_field_errors %}class="error"{% endif %}>
+								{{ form.field }}
+								<span>{{ form.instance.field.name }}</span>
+								{{ form.field.errors }}
+							</td>
+							<td {% if form.width.errors or form.non_field_errors %}class="error"{% endif %}>
+								{{ form.width }}
+								{{ form.width.errors }}
+							</td>
+							<td {% if form.ordering.errors or form.non_field_errors %}class="error"{% endif %}>
+								{{ form.ordering }}
+								{{ form.ordering.errors }}
+							</td>
+							<td>
+								{% if form.instance.pk %}
+									{{ form.id }}
+									{{ form.DELETE }}
+								{% endif %}
+							</td>
+						</tr>
+					{% endfor %}
+				</table>
+				{{ formset.management_form }}
+			
+			</div>
+        </div>
+        
+        <ul id="column_options" style="width: 200px;" class="dropdown-menu"></ul>
```

Comment:
Lines 230 through 243 render a confirmation screen with a link to the PDF. In the current implementation, the PDF downloads to my downloads folder, but there's no other feedback to the user...which could be confusing. Could you resurrect some kind of confirmation screen?